packages/infobox/stories/InfoBox.stories.tsx

```diff
@@ -15,20 +16,19 @@ export default {
     },
     className: {
       control: { disable: true }
+    },
+    message: {
+      control: { type: "text" },
```

Comment:
I thought storybook was smart enough to handle default controls by now but this does help so that we don't need to wrap the text in a string in the input.